apps/ml-service/simple_retrain.py

```
@@ -0,0 +1,42 @@
+sys.path.append(os.path.join(os.path.dirname(__file__), 'src'))
+from models.train import retrain_models_with_recent_data
+def main():
+    """Simple retraining test"""
+    print("🚀 Testing Model Retraining with Current Data")
+    print("=" * 50)
+    
+    try:
+        # Use existing training infrastructure
+        results = retrain_models_with_recent_data()
+        
+        print(f"\n✅ Training Results:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)